src/main/java/website/frontrow/music/MusicPlayer.java

```diff
@@ -0,0 +1,95 @@
+package website.frontrow.music;
+
+import javafx.embed.swing.JFXPanel;
+import javafx.scene.media.Media;
+import javafx.scene.media.MediaPlayer;
+import website.frontrow.logger.Log;
+
+/**
+ * Class for playing music files.
+ */
+@SuppressWarnings("checkstyle:hideutilityclassconstructor")
+/* The constructor is used to play the opening song and to make sure JavaFX is running. */
```

Comment:
The warning is pretty much correct in that you are doing something wrong.
Don't do this.